site/index.html

```diff
@@ -79,8 +82,16 @@
             <div class="row">
                 <div class="col-3"></div>
                 <div class="col-6">
-                    <label for="selected-area">Test suite:&nbsp;</label>
-                    <select id="selected-area" name="selected-area"></select>
+                    <label for="test-selected-area">Test suite:&nbsp;</label>
```

Comment:
I think it would look nice if we can align the labels and the drop-downs of Test suite and Date filters. i.e
instead of having both <label> and <select> in the same 'col-6' div, we can instead put them in their own divs, with labels inside 'col-2' and the selects inside 'col-4'